src/utils/dotnetDevCertsHttps.ts

```diff
@@ -5,23 +5,18 @@
 
 import * as cp from 'child_process';
 import { getExtensionPath } from '../common';
+import { commonOptions, omnisharpOptions } from '../shared/options';
 
 // Will return true if `dotnet dev-certs https --check` succesfully finds a trusted development certificate.
-export async function hasDotnetDevCertsHttps(dotnetPath: string): Promise<ExecReturnData> {
-    return await execChildProcess(
-        `${dotnetPath ? dotnetPath : 'dotnet'} dev-certs https --check --trust`,
-        process.cwd(),
-        process.env
-    );
+export async function hasDotnetDevCertsHttps(): Promise<ExecReturnData> {
+    const dotnetPath = (commonOptions.useOmnisharpServer ? omnisharpOptions.dotnetPath : undefined) ?? 'dotnet';
+    return await execChildProcess(`${dotnetPath} dev-certs https --check --trust`, process.cwd(), process.env);
 }
 
 // Will run `dotnet dev-certs https --trust` to prompt the user to create a trusted self signed certificates. Retruns true if sucessfull.
-export async function createSelfSignedCert(dotnetPath: string): Promise<ExecReturnData> {
-    return await execChildProcess(
-        `${dotnetPath ? dotnetPath : 'dotnet'} dev-certs https --trust`,
-        process.cwd(),
-        process.env
-    );
+export async function createSelfSignedCert(): Promise<ExecReturnData> {
+    const dotnetPath = (commonOptions.useOmnisharpServer ? omnisharpOptions.dotnetPath : undefined) ?? 'dotnet';
```

Comment:
Maybe extract this into a function where it can be written in a more readable form.